File: app/src/main/java/com/viafourasample/src/activities/commentsContainer/CommentsContainerViewModel.java
package com.viafourasample.src.activities.commentsContainer;

import com.viafourasample.src.managers.StoryManager;
import com.viafourasample.src.model.Story;

public class CommentsContainerViewModel {
    private Story story;

    public CommentsContainerViewModel(String containerId) {
        this.story = StoryManager.getInstance().getStory(containerId);
    }
    public Story getStory() {
        return story;
    }
}

File: app/src/main/java/com/viafourasample/src/activities/commentsContainer/CommentsContainerViewModel.kt
package com.viafourasample.src.activities.commentsContainer

import com.viafourasample.src.managers.StoryManager
import com.viafourasample.src.model.Story

class CommentsContainerViewModel(containerId: String) {
    val story: Story = StoryManager.getInstance().getStory(containerId)
}
